go.mod

```diff
@@ -65,10 +65,10 @@ require (
 	github.com/go-openapi/validate v0.18.0
 	github.com/go-ozzo/ozzo-validation v3.5.0+incompatible // indirect
 	github.com/godbus/dbus v0.0.0-20151105175453-c7fdd8b5cd55
-	github.com/gogo/protobuf v1.2.0
+	github.com/gogo/protobuf v0.0.0-20171007142547-342cbe0a0415
```

Comment:
Please don't update any dependencies. Per slack discussion you really don't need any of the go.mod/go.sum changes.

Reply:
I won't, i promise :-)